src/Services/SchemaPoster.php

```diff
@@ -43,18 +46,19 @@ class SchemaPoster {
   /**
    * Constructor.
    *
-   * @param \Drupal\Core\Logger\LoggerChannelFactoryInterface $loggerChannelFactory
+   * @param \Drupal\Core\Logger\LoggerChannelFactoryInterface $logger_channel_factory
    *   Injected when called as a service.
-   * @param \Drupal\search_api_pantheon\Services\PantheonGuzzle $pantheonGuzzle
+   * @param \Drupal\search_api_pantheon\Services\PantheonGuzzle $pantheon_guzzle_client
    *   Injected when called as a service.
    */
-  public function __construct(LoggerChannelFactoryInterface $loggerChannelFactory, PantheonGuzzle $pantheonGuzzle) {
-    $this->logger = $loggerChannelFactory->get('PantheonSolr');
-    $this->client = $pantheonGuzzle;
+  public function __construct(LoggerChannelFactoryInterface $logger_channel_factory, PantheonGuzzle $pantheon_guzzle_client) {
```

Comment:
As per Drupal's var name best practices, all var and arg names are in snake case, while class properties are in camelcase.